packages/tradershub/src/screens/TermsOfUse/TermsOfUse.tsx

```diff
@@ -1,24 +1,36 @@
 import React from 'react';
 import { Form, Formik, FormikValues } from 'formik';
+import { Text } from '@deriv-com/ui';
 import Actions from '../../flows/RealAccountSIgnup/SignupWizard/Actions';
 import WizardScreenWrapper from '../../flows/RealAccountSIgnup/SignupWizard/WizardScreenWrapper';
 import { ACTION_TYPES, useSignupWizardContext } from '../../providers/SignupWizardProvider/SignupWizardContext';
+import FatcaDeclaration from './TermsOfUseSections/FatcaDeclaration';
+import PEPs from './TermsOfUseSections/PEPs';
+
+// Temporary till we have this in deriv-com/ui
```

Comment:
Append TODO: infront of any sort of comment or changes:
`TODO: Replace divider from deriv-com/ui once it is implemented`

Reply:
Sonarcloud will complain if i add TODO